README.md

```diff
@@ -61,15 +61,15 @@ Here's a basic example of what map configuration files look like:
 
 ## Compiling
 
-1. Clone the repo to your computer.
+1. Clone the repo to your computer. `git clone https://github.com/Warzone/TGM`
```

Comment:
Docs changes should be allotted to a different PR if they aren't relevant to the main change.